dotcom-rendering/src/client/userFeatures/index.ts

```diff
@@ -0,0 +1,5 @@
+import { refresh } from './user-features';
+
+export const userFeatures = async (): Promise<void> => {
+	await refresh(); // come to Promise structure
+};
```

Comment:
[Our code style recommends against `index.ts` files](https://github.com/guardian/dotcom-rendering/blob/1fc26de02112dc29a1273036d974448d92d514d9/dotcom-rendering/docs/contributing/code-style.md?plain=1#L51) – thanks @cemms1.

Prefer exporting/importing `refresh` directly.

Reply:
As discussed, @guardian/dotcom-platform should make sure that this recommendation is followed by the current code, as we have examples that go against this recommendation all over :(